eng/common/instructions/azsdk-tools/local-sdk-workflow.instructions.md

```diff
@@ -0,0 +1,119 @@
+---
+description: "Guide the user to generate and build SDKs locally for TypeSpec changes"
+---
+
+# Goal
+
+Help the user generate and build SDKs locally from TypeSpec API specifications using the `azure-sdk-mcp` tools.
```

Comment:
May I know how customers would refer these instructions? I may not quite understand the context here, are these instructions used in customers' IDEs?